data/isdialogmote/innkalling-arbeidstaker.json

```diff
@@ -1,7 +1,47 @@
-{
-  "innkalling": {
-      "tidOgSted": {
-      "sted": "Innkallingsgata 4"
-    }
+[{
+    "header": "Møtetidspunkt"
+  },
+  {
+    "ptext": "20. mai 2021, Storgata 4"
+  },
+  {
+    "header": "Møtested"
+  },
+  {
+    "ptext": "Videomøte på Teams"
+  },
+  {
+    "header": "Lenke til videomøte"
+  },
+  {
+    "link": "https://teams.microsoft.com/l/osv.osv.osv"
+  },
+  {
+    "ptext": "Hei Artig Trane"
+  },
+  {
+    "ptext": "Velkommen til dialogmøte mellom deg, arbeidsgiveren din og en veileder fra NAV. I møtet skal vi snakke om situasjonen din og bli enige om en plan som kan hjelpe deg videre."
+  },
+  {
+    "ptext": "Her kommer en fritekst skrevet av veilederen."
+  },
+  {
+    "ptext": "Den som har sykmeldt deg, eller en annen behandler, kan også bli invitert til å delta i møtet. Til dette møtet har vi ikke sett behov for det."
+  },
+  {
+    "header": "Før møtet"
+  },
+  {
+    "ptext": "Det er viktig at dere fyller ut oppfølgingsplanen sammen og deler den med NAV. Det gir oss et godt utgangspunkt for å snakke om hva som fungerer, hva som har blitt forsøkt, og hvilke muligheter som finnes framover."
+  },
+  {
```

Comment:
Denne delen får for store mellomrom mellom linjene, så kanskje vi skulle innføre en egen type signaturtype?

Reply:
Ja, det kan nok fungere 👍 Ellers kan vi ha newlines, hvis vi klarer å få de med hele veien 🤔

Reply:
En mulighet er kanskje å tillate flere text-felter i en PARAGRAPH, og la det bety at de skal rendres i en <p>.. </p> med <br> i mellom hver text?